src/profiles/idw/yaml_helper.py

```diff
@@ -141,21 +142,10 @@ def __init__(
         yaml.constructor.SafeConstructor.add_constructor(
             path_tag, self._path_constructor
         )
-        yaml.add_representer(Path, self._path_reprer)
+        yaml.add_representer(PosixPath, self._path_reprer)
```

Comment:
Just wondering why we need to use `PosixPath` specifically here? `PosixPath` cannot be instantiated on Windows, and while Windows isn't our primary OS, it's nice to keep that portability if possible.

Reply:
good point! But the ingestion script will be running on a server which is a Linux system, and this wouldn't affect the YAML file, which might be on a Windows machine, so should be fine? PS: could test on a Windows machine if needed.